### ISSUE TITLE
Can we have the APK directly in the github repository instead of the source code provided ?

### ISSUE THREAD

**Issue by saurabhlucideus** [oversecured/ovaa#3]
It will be great if we can directly access the APK instead of building the application using the source code.

**Comment by enovella** [comment]
It could be done with releases via CI/CD or manually after some stability.

**Comment by Sahabalam** [comment]
yeah,there is a problem in building this .apk in android sharing because of ndk version. so i did you can download this one.


https://drive.google.com/file/d/1x1wxM0FNQN4rgxiTeB3bKbiFuJWdr0_V/view?usp=sharing

**Comment by blackf0x** [comment]
It's not ovaa apk application :/

**Comment by xskullboyx** [comment]
ovaa apk direct link: https://drive.google.com/file/d/1Xbjok3SmYbs9TBjePUoQtpJFv7g4AeYU/view?usp=share_link

**Comment by rew1nter** [comment]
How did you guys build it?

**Comment by itis911** [comment]
> How did you guys build it?

download it directly from :  https://drive.google.com/file/d/1x1wxM0FNQN4rgxiTeB3bKbiFuJWdr0_V/view?usp=sharing

**Comment by kbamol** [comment]
> yeah,there is a problem in building this .apk in android sharing because of ndk version. so i did you can download this one.
> 
> https://drive.google.com/file/d/1x1wxM0FNQN4rgxiTeB3bKbiFuJWdr0_V/view?usp=sharing
I guess this one is not OVAA Apk when I decompile it shows some awesome project apk

**Comment by Tig3rHu** [comment]
> > yeah,there is a problem in building this .apk in android sharing because of ndk version. so i did you can download this one.
> > https://drive.google.com/file/d/1x1wxM0FNQN4rgxiTeB3bKbiFuJWdr0_V/view?usp=sharing
> > I guess this one is not OVAA Apk when I decompile it shows some awesome project apk

yes，i guess so

**Comment by Tig3rHu** [comment]
I rebuilt the environment to compile ovaa, which is a usable apk : https://github.com/Tig3rHu/ovaa_apk_security

**Comment by moolakarapaiyan** [comment]
its is very easy to build by ourselves
open android studio
new -> project from version control -> paste this GitHub url

got an error while building this , solved by referring stack overflow page
https://stackoverflow.com/questions/54273412/failed-to-install-the-following-android-sdk-packages-as-some-licences-have-not

**Comment by loulan-ling** [comment]
Before compiling the APK, You need to set android:usesCleartextTraffic="true" in AndroidManifest.xml, otherwise your traffic can only go through https.
